roles/openshift_logging_defaults/defaults/main.yml

```diff
@@ -1,7 +1,7 @@
 ---
 l_openshift_logging_search_dict:
   origin: "${component}"
-  openshift-enterprise: "ose-${component}"
+  openshift-enterprise: "${component}"
```

Comment:
@ewolinetz a number of components utilize 'l_openshift_logging_search' which is based on this dictionary.  Can you verify they all compatible with this change?

If they are, we can get rid of l_openshift_logging_search_dict and l_openshift_logging_search; replace l_openshift_logging_search in various places with "${component}" directly.

This includes most of logging and metrics.

The description of the PR doesn't match

Reply:
@michaelgugino I didn't realize that `openshift_metrics` depended on this dict too... that seems like scope creep that we should fix. I know that this change resolves logging specific issues for 3.11+ since we have updated our image names to include `ose-` in the name (WRT the base components).

@richm are we following this as well for the event-router image?

Reply:
We are not doing this for the eventrouter image

Reply:
@michaelgugino does the latest commit reflect what you're suggesting? If so, I can squash commits